### DISCUSSION TITLE
Miryoku with custom combos for German Umlauts "Ä", "Ö", "Ü", and "ß"

### DISCUSSION THREAD

**Discussion by 10bn** [manna-harbour/miryoku#364] [Show and tell]
If you love Miryoku's efficiency but need easy access to German umlauts like "Ä", "Ö", "Ü", and "ß", these custom combos might be for you. They’re built around the EuroKey layout but can be easily adapted for the US International layout if needed. 

https://gist.github.com/10bn/7ac5e9f0a0e74360e4b4b032d00364a6

**Comment by manna-harbour** [comment]
Thanks for sharing your combos, it looks like a nice solution!
